### PR TITLE
Adding Form and footer

### PR DESCRIPTION
**Pull request by mzirkof**
### Adding Form & Footer to Portfolio
:magic_wand:  Changes made here are the following (not necessarily in this order but they are managed in this pull request )

  :heavy_check_mark:  Added the contact form (linked with the Formspree system which collects data send)
  :heavy_check_mark:  Added The footer containing social media links 
  :heavy_check_mark:  Added decorative background images 
  :heavy_check_mark:  Containers classes and children use both grid and flex display property
  :heavy_check_mark:  Buttons have the hover property
  :heavy_check_mark:  All changes are positioned and styled by updating the styles.css file )
  :heavy_check_mark:  Thanks to @pulgosodog  for his useful contribution to fixing some issues on buttons and the form section ( :laughing: I learn a good tip about sizing ) 

:pray:  Hope to get a lot of tips and advice from reviewers thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -212,7 +212,44 @@ <h1>
...
+  <section class="form_s">
+    <h1>
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h1>
+    <form class="form" action="https://formspree.io/f/xlekzalj" method="POST">
+      <fieldset>
+        <ul>
+          <li><input type="text" name="user_name" placeholder="Full name" required ></li>
+          <li><input type="email" name="user_mail" placeholder="Email address" required ></li>
+          <li><textarea name="user_msg" id="msg" cols="30" rows="10" required>Hey I'd like to connect to...</textarea></li>
```
Comment (Owusu-Desmond) [inline index.html:228]: - [ ] Please from the project requirement, the textarea field should accept up to 500 characters so I think you should add the `maxlength` attribute for the textarea field and make it accept 500 chars :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @mzirkof and @pulgosodog , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -212,7 +212,44 @@ <h1>
...
+
+  <!-- Form section of Portfolio -->
+  <section class="form_s">
+    <h1>
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h1>
+    <form class="form" action="https://formspree.io/f/xlekzalj" method="POST">
+      <fieldset>
+        <ul>
+          <li><input type="text" name="user_name" placeholder="Full name" required ></li>
```
[the remark this answers is not in the document]
Comment (mzirkof) [reply index.html:226]: Ok done

File: index.html (hunk 1 above)
Comment (mzirkof) [reply index.html:228]: Ok done

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @mzirkof , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- You implemented the requested changes successfully. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**